### PR TITLE
Use Flask-Migrate in `Makefile`

### PR DESCRIPTION
**Pull request by michplunkett**
## Description of Changes
If we are using Flask-Migrate commands everywhere else, we shouldn't make an exception for the `Makefile`.

## Tests and Linting
- [x] This branch is up-to-date with the `develop` branch.
- [x] `pytest` passes on my local development environment.
- [x] `pre-commit` passes on my local development environment.
- [x] Verify `make create_db` works correctly.

```console
(env) michaelp@MacBook-Air-18 OpenOversight % make create_db
docker compose build
WARN[0000] The "AWS_DEFAULT_REGION" variable is not set. Defaulting to a blank string.
WARN[0000] The "AWS_SECRET_ACCESS_KEY" variable is not set. Defaulting to a blank string.
WARN[0000] The "AWS_ACCESS_KEY_ID" variable is not set. Defaulting to a blank string.
WARN[0000] The "S3_BUCKET_NAME" variable is not set. Defaulting to a blank string.
WARN[0000] The "APPROVE_REGISTRATIONS" variable is not set. Defaulting to a blank string.
[+] Building 0.6s (25/25) FINISHED                                                                                                                                                         docker:desktop-linux
 ...
Postgres is up
## Creating database
docker compose run --rm web flask db stamp head
...
INFO  [sqlalchemy.engine.Engine] COMMIT
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
(env) michaelp@MacBook-Air-18 OpenOversight %
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ create_db: start
 	done
 	@echo "Postgres is up"
 	## Creating database
-	docker compose run --rm web alembic --config=./OpenOversight/migrations/alembic.ini stamp head
+	docker compose run --rm web flask db stamp head
```
Comment (michplunkett) [inline Makefile:32]: We don't specifically reference Alembic anywhere else in the project. This makes the `Makefile` consistent with the rest of the app.

**Review by sea-kelp — APPROVED** [review]
Looks good to me! It might also be worth adding a make target for Flask-Migrate:
https://github.com/OrcaCollective/OpenOversight/blob/6d5c7b08b24a7669218d78b780f6547132d89351/justfile#L95-L97

**Comment by michplunkett** [comment]
I'll take a look and see what that looks like for Make.

**Comment by michplunkett** [comment]
Looks like that functionality is a little more complicated with `Makefile`s: https://stackoverflow.com/a/45003119

**Pull request merged**